dags/parquet_import.py

```diff
@@ -0,0 +1,61 @@
+import os
+from datetime import datetime
+from airflow.models import DAG
+from airflow.operators.python import PythonOperator
+from airflow.providers.postgres.operators.postgres import PostgresOperator
+
+def get_latest_month_year():
+    currentMonth = datetime.now().month - 1
```

Comment:
Instead of `datetime.now()`, should we use the logical execution date instead? Otherwise, backfilling (importing data older than the last month) won't be possible.